src/stores/root.ts

```diff
@@ -1618,6 +1625,21 @@ export class RootStore {
 					},
 				],
 			},
+            {
+				poolId: '697',
```

Comment:
please ensure the spot price of this pool is appropriate

Reply:
@JeremyParish69 
current spot price is correct.:)